daisy/compute/test_client.go

```diff
@@ -99,6 +100,9 @@ type TestClient struct {
 	SetCommonInstanceMetadataFn func(project string, md *compute.Metadata) error
 	RetryFn                     func(f func(opts ...googleapi.CallOption) (*compute.Operation, error), opts ...googleapi.CallOption) (op *compute.Operation, err error)
 
+	// Beta API calls
+	GetGuestAttributesFn func(project, zone, name, queryPath, variableKey string) (*computeBeta.GuestAttributes, error)
```

Comment:
minor: how about calling this `GetGuestAttributesOverride` since its purpose is to override the default impl?

Reply:
Maybe, but lines 51-100 :)